src/code/z_kaleido_setup.c

```diff
@@ -1,22 +1,26 @@
-#include "global.h"
+#include "z64pause_menu.h"
+
+#include "z64.h"
 #include "z64rumble.h"
 #include "z64shrink_window.h"
-#include "z64view.h"
+
 #include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
 
-s16 sKaleidoSetupRightPageIndex[] = {
+s16 sKaleidoSetupRightPageIndex[PAUSE_PAGE_MAX - 1] = {
```

Comment:
tbh I think using max-1, since the array is one short, defeats the point
and now I wonder why these arrays even have only 4 members :thinking:

Reply:
At first I thought the array was being indexed from element 1, but I misread. I'll undo this